src/Components/Facility/TriageForm.tsx

```diff
@@ -326,99 +288,64 @@ export const TriageForm = (props: triageFormProps) => {
-                  <InputLabel id="num-patients-isolation-label">
-                    Suspected Isolated
-                  </InputLabel>
-                  <LegacyTextInputField
+                  <TextFormField
                     name="num_patients_isolation"
-                    variant="outlined"
-                    margin="dense"
                     type="number"
-                    InputLabelProps={{
-                      shrink: !!state.form.num_patients_isolation,
-                    }}
+                    label="Suspected Isolated"
                     value={state.form.num_patients_isolation}
-                    onChange={handleChange}
-                    errors={state.errors.num_patients_isolation}
+                    onChange={handleFormFieldChange}
+                    error={state.errors.num_patients_isolation}
                   />
                 </div>
                 <div>
-                  <InputLabel id="num-patient-referred-label">
-                    Patients Referred
-                  </InputLabel>
-                  <LegacyTextInputField
+                  <TextFormField
                     name="num_patient_referred"
-                    variant="outlined"
-                    margin="dense"
                     type="number"
-                    InputLabelProps={{
-                      shrink: !!state.form.num_patient_referred,
-                    }}
+                    label="Patients Referred"
                     value={state.form.num_patient_referred}
-                    onChange={handleChange}
-                    errors={state.errors.num_patient_referred}
+                    onChange={handleFormFieldChange}
+                    error={state.errors.num_patient_referred}
                   />
                 </div>
                 <div>
-                  <InputLabel id="num-patient-referred-label">
-                    Confirmed Positive
-                  </InputLabel>
-                  <LegacyTextInputField
+                  <TextFormField
                     name="num_patient_confirmed_positive"
-                    variant="outlined"
-                    margin="dense"
                     type="number"
-                    InputLabelProps={{
-                      shrink: !!state.form.num_patient_confirmed_positive,
-                    }}
+                    label="Confirmed Positive"
                     value={state.form.num_patient_confirmed_positive}
-                    onChange={handleChange}
-                    errors={state.errors.num_patient_confirmed_positive}
+                    onChange={handleFormFieldChange}
+                    error={state.errors.num_patient_confirmed_positive}
                   />
                 </div>
               </div>
               <div className="flex flex-col md:flex-row gap-2 justify-between mt-4">
                 <Cancel onClick={() => goBack()} />
                 <Submit onClick={handleSubmit} label={buttonText} />
```

Comment:
Wouldn't the `handleSubmit` specified in the `form` component be enough?

```suggestion
                <Submit label={buttonText} />
```